docs/software/sciapps/gromacs.md

```diff
@@ -1,4 +1,161 @@
 [](){#ref-uenv-gromacs}
 # GROMACS
-!!! todo
-    complete docs
+
+[GROMACS](https://www.gromacs.org) (GROningen Machine for Chemical Simulations) is a versatile and widely-used open source package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
+
+It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since GROMACS is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers.
+
```

Comment:
Can you please add the note like in the other sciapps pages mentioning uenv and linking to the appropriate documentation?